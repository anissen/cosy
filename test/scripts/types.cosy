var b1 = true
print b1
var b2 = false
print b2

var i1 = 100
print i1
var i2 = -7.5
print i2

var s1 = 'hello'
print s1

var f1 = fn () {}
print f1
var f2 = fn (xyz) { print xyz }
print f2
