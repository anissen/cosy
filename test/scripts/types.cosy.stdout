true
false
100
-7.5
hello
<fn>
<fn>
